### PR TITLE
Update previous activity cost tables [skip deploy]

### LINKED ISSUE #1111: Implement style changes & reduce content for Actual Costs of Prior Activities table
Moving forward from design in #1099 

Update the "[Actual Costs](https://hitech-apd.app.cloud.gov/apd#prev-activities-table)" of Prior Activities tables by:
- Removing the columns shown in both tables
- Adding the column shown in the MMIS table
- Changing the cell labels as shown in both tables
- Restyling the table title from a row to `h3` to match elsewhere

(Click the images to enlarge and make them legible 👇 )
screen shot 2018-11-08 at 6 07 02 pm

screen shot 2018-11-08 at 6 06 55 pm

To match this layout when implemented:
screen shot 2018-11-08 at 4 24 32 pm

### This task is done when...
 - [ ] The HIT + HIE table matches the mockup
 - [ ] The MMIS table matches the mockup

### PR DESCRIPTION
**Pull request by mgwalker**
Closes #1111

Can't deploy this one because it depends on backend changes, and I didn't push that through first.  😬

localhost_8080_apd

### This pull request is ready to merge when...
- [ ] Tests have been updated (and all tests are passing)
- [ ] This code has been reviewed by someone other than the original author

### This feature is done when...
- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: api/migrations/20181115190951_update-previous-activities.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
+exports.up = async knex => {
+  await knex.schema.alterTable('apd_previous_activity_expenses', table => {
```
Comment (mgwalker) [inline api/migrations/20181115190951_update-previous-activities.js]: Adds new columns, computes their values from the existing columns, and then removes the old columns.

File: web/src/containers/ApdPreviousActivityTableMMIS.js [hunk 2]
```diff
@@ -5,277 +5,225 @@ import { connect } from 'react-redux';
...
+                  0
+                );
+                const actual = [90, 75, 50].reduce(
+                  (total, ffp) => total + expenses[ffp].federalActual,
+                  0
+                );
+
+                return (
+                  <tr key={year} className="align-middle">
+                    <th id={`prev_act_mmis_row_${year}`}>
```
Comment (mgwalker) [inline web/src/containers/ApdPreviousActivityTableMMIS.js]: Why is this thing so tall?!

File: web/src/constants.js [hunk 3]
```diff
@@ -19,3 +21,15 @@ export const DASHBOARD_TASK_BUTTON_TEXT = {
   [APD_STATUS.DISAPPROVED]: 'Disapproval Letter',
   [APD_STATUS.WITHDRAWN]: 'View'
 };
+
+export const TABLE_HEADERS = {
```
Comment (hbillings) [inline web/src/constants.js]: aw yiss

File: web/src/styles/app/table.css [hunk 4]
```diff
@@ -146,4 +150,5 @@
 .table-frozen-narrow-header .table-frozen-data tr > th:first-child,
 .table-frozen-narrow-header .table-frozen-data tr > td:first-child {
   min-width: 90px;
+  width: 90px;
```
Comment (mgwalker) [inline web/src/styles/app/table.css]: I honestly can't remember why I needed this.  But if I didn't have it, the hidden columns were too wide.  They'd just explode in size, and the second column would be hidden by it forever.  It was weird.

**Comment by hbillings** [comment]
I'm guessing the last one is probably rendering inputs for spacing when it shouldn't be. Throw some time on my cal for tomorrow and we'll figure it out!

**Pull request merged**